librz/analysis/p/analysis_v810.c

```diff
@@ -429,17 +441,49 @@ static char *get_reg_profile(RzAnalysis *analysis) {
 		"gpr	r29	.32	116 0\n"
 		"gpr	r30	.32	120 0\n"
 		"gpr	r31	.32	124 0\n"
-		"gpr	pc	.32	128 0\n"
+		"gpr	PC	.32	128 0\n"
+
+		"gpr	EIPC	.32	132	0\n"
+		"gpr	EIPSW	.32	136	0\n"
+		"gpr	FEPC	.32	140	0\n"
+		"gpr	FEPSW	.32	144	0\n"
+		"gpr	ECR	.32	148	0\n"
+		"gpr	PSW	.32	152	0\n"
+		"gpr	PIR	.32	156	0\n"
+		"gpr	TKCW	.32	160	0\n"
+		"gpr	Reserved_8	.32	164	0\n"
+		"gpr	Reserved_9	.32	168	0\n"
+		"gpr	Reserved_10	.32	172	0\n"
+		"gpr	Reserved_11	.32	176	0\n"
+		"gpr	Reserved_12	.32	180	0\n"
+		"gpr	Reserved_13	.32	184	0\n"
+		"gpr	Reserved_14	.32	188	0\n"
+		"gpr	Reserved_15	.32	192	0\n"
+		"gpr	Reserved_16	.32	196	0\n"
+		"gpr	Reserved_17	.32	200	0\n"
+		"gpr	Reserved_18	.32	204	0\n"
+		"gpr	Reserved_19	.32	208	0\n"
+		"gpr	Reserved_20	.32	212	0\n"
+		"gpr	Reserved_21	.32	216	0\n"
+		"gpr	Reserved_22	.32	220	0\n"
+		"gpr	Reserved_23	.32	224	0\n"
```

Comment:
why these?

Reply:
Since several instructions need to read and write SR registers with index, these reversed registers need to be at the location.